tools/cldr-apps/js/src/esm/cldrAnnounce.mjs

```diff
@@ -119,6 +149,7 @@ async function compose(formState, viewCallbackComposeResult) {
 }
 
 async function saveCheckmark(checked, announcement) {
+  resetSchedule();
```

Comment:
this fixes a bug -- otherwise, the "unread" number by the balloon icon in the main menu took 60 seconds before it updated in response to adding a checkmark for "I have read this"